libraries/AP_SerialManager/AP_SerialManager.cpp

```diff
@@ -424,6 +551,21 @@ void AP_SerialManager::init_console()
 // init - // init - initialise serial ports
 void AP_SerialManager::init()
 {
+    if (enable_ext_serial) {
```

Comment:
needs #if wrapper

Reply:
Yes that's the plan, once I have it all tested and verified